semgrep-core/core/Metavariable.ml

```diff
@@ -15,9 +15,15 @@
  * license.txt for more details.
 *)
 (*e: pad/r2c copyright *)
+open Printf
 open Common
 module G = AST_generic
 
+(* Provide hash_* and hash_fold_* for the core ocaml types *)
+open Ppx_hash_lib.Std.Hash.Builtin
```

Comment:
This is annoying ... deriving eq and show do not need this.

Reply:
I'm not sure why I had to resort to this. It seems wrong indeed.